source/javascripts/components/unified-editor/WorkflowConfig/tabs/PropertiesTab.tsx

```diff
@@ -1,11 +1,12 @@
 import { ChangeEventHandler, useEffect, useState } from 'react';
-import { Box, Textarea } from '@bitrise/bitkit';
+import { Box, Button, Textarea } from '@bitrise/bitkit';
 import { useDebounceCallback } from 'usehooks-ts';
 import useBitriseYmlStore from '@/hooks/useBitriseYmlStore';
 import WorkflowService from '@/core/models/WorkflowService';
 import useFeatureFlag from '@/hooks/useFeatureFlag';
 import EditableInput from '@/components/EditableInput/EditableInput';
 import useRenameWorkflow from '@/components/unified-editor/WorkflowConfig/hooks/useRenameWorkflow';
+import { useWorkflowsPageStore } from '@/pages/WorkflowsPage/WorkflowsPage.store';
```

Comment:
In my opinion, it is better if we don't connect this store to a component that is used not just on the workflows page.